demisto_sdk/commands/init/initiator.py

```diff
@@ -413,6 +451,12 @@ def integration_init(self) -> bool:
         self.copy_common_server_python()
         self.copy_demistotmock()
 
+        if self.template != self.DEFAULT_INTEGRATION_TEMPLATE:  # DEFAULT_INTEGRATION_TEMPLATE there are no secrets
+            ignore_secrets = input("\nCurrently, there are some secrets in the pack. "
+                                   "Would you like ignore them automatically? Y/N ").lower()
+            if ignore_secrets in ['y', 'yes']:
```

Comment:
2 notes:
1. do lower case on `ignore_secrets` to accept Y or YES and such.
2. Please print the detected secrets to the user - so he'd know what he is ignoring.